### PR TITLE
BP-462: Uniform styling EventType-/EventQuery-/Mapping-CreationView

### PR DESCRIPTION
**Pull request by julianweise**

### CODE REVIEW

File: src/CreateEventQueryView/CreateEventQueryView.js [hunk 1]
```diff
@@ -154,30 +154,36 @@ class CreateEventQueryView extends ConnectionComponent {
...
+							</Col>
+							<Col md={4} style={{paddingTop: 20}}>
+								<EventTypeInformation attributes={attributes}/>
+							</Col>
+						</Row>
+						<Row>
+							<Col md={8}>
+								<div className={css(AppStyles.textAlignCenter)}>
+									{this.getAbortButton()}
+									<RaisedButton
```
Comment (frederike-ramin) [inline src/CreateEventQueryView/CreateEventQueryView.js:176]: I'd really prefer the buttons to be in the center of the whole page.

File: src/CreateEventQueryView/EventQueryInputArea.js [hunk 2]
```diff
@@ -60,9 +61,11 @@ class EventQueryInputArea extends Component {
 					data-hint={help.input.eventQueryView.query}
 					data-hintPosition="middle-right"
 					floatingLabelText={config.descriptions.queryInputFieldHint + this.getNotEditablePartOfEventQuery()}
+					floatingLabelStyle={{color: config.colors.primaryDarkAlphaDarker}}
 					name="event-query"
 					hintText={config.descriptions.exampleQuery}
 					multiLine={true}
+					rows={5}
```
Comment (frederike-ramin) [inline src/CreateEventQueryView/EventQueryInputArea.js]: Maybe reduce that to 3 or something? The Area is really huge.

File: src/CreateEntityMappingView/CreateEntityMappingView.js [hunk 3]
```diff
@@ -391,12 +396,14 @@ class CreateEntityMappingView extends ConnectionComponent {
 		}
 		return (
 			<Container>
+				<Paper style={{paddingLeft: 20, paddingRight: 20, marginTop: 40, paddingTop: 20}} zDepth={2}>
 				<div>
 					{this.renderSelectTargetStatus()}
 					{this.renderSelectTypes(initialData)}
 					{initialData.attributeFields}
 					{CreateEntityMappingView.renderButtons(submitCallback)}
```
Comment (frederike-ramin) [inline src/CreateEntityMappingView/CreateEntityMappingView.js:404]: The buttons are not centered are they? Seems like the submit button is too far on the right.

File: src/AppStyles.js [hunk 4]
```diff
@@ -117,10 +117,21 @@ const AppStyles = StyleSheet.create({
 		color: 'white'
 	},
 	
+	textLabel: {
```
Comment (frederike-ramin) [inline src/AppStyles.js]: That's some kind of error label right?

File: src/CreateEventQueryView/EventTypeInformation.js [hunk 5]
```diff
@@ -3,11 +3,13 @@ import {List, ListItem} from "material-ui/List";
 import {css} from "aphrodite";
 import {Card, CardText, CardTitle} from "material-ui";
 import AppStyles from "../AppStyles";
+import FilterList from 'material-ui/svg-icons/device/widgets';
```
Comment (frederike-ramin) [inline src/CreateEventQueryView/EventTypeInformation.js]: Maybe rename this to FilterListIcon. This name is pretty confusing.

**Review by frederike-ramin — CHANGES_REQUESTED** [review]

File: src/CreateEventQueryView/CreateEventQueryView.js (hunk 1 above)
Comment (julianweise) [reply src/CreateEventQueryView/CreateEventQueryView.js:176]: rejected

File: src/CreateEntityMappingView/CreateEntityMappingView.js (hunk 3 above)
Comment (julianweise) [reply src/CreateEntityMappingView/CreateEntityMappingView.js:404]: They are centered -rejected

Comment (frederike-ramin) [reply src/CreateEntityMappingView/CreateEntityMappingView.js:404]: No. They are not.

**Review by frederike-ramin — APPROVED** [review]

**Pull request merged**